### PR TITLE
point robo-dotd log to /tmp/

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: bin/dotd [hunk 1]
```diff
@@ -22,7 +22,8 @@ class String
   end
 end
 
-LOG_FILE = '/var/log/code-dot-org-dotd.log'
+BIN_DIR = File.expand_path(File.dirname(__FILE__))
+LOG_FILE = "#{BIN_DIR}/log/dotd.log"
```
Comment (ashercodeorg) [inline bin/dotd]: Does this "work" if the user has made a local copy of the script? IIRC, this practice was encouraged, as the script is (intentionally) dependency free.

**Comment by ashercodeorg** [comment]
Other than comment, LGTM.

**Comment by islemaster** [comment]
LGTM

File: bin/dotd (hunk 1 above)
Comment (Hamms) [inline bin/dotd]: Hmm, unfortunately probably not; I had forgotten that was an intended use case. We could store the logfile alongside the script, which would put it in the `cdo/bin` directory for typical use case, but I'm not positive I like that.

Alternatively, we could go back to just dropping it in `/tmp/`

**Comment by Hamms** [comment]
PTAL

**Comment by islemaster** [comment]
It looks like [`/tmp` _should_ be appropriate](http://apple.stackexchange.com/questions/210942/how-to-determine-public-temp-directory-on-os-x-to-be-write-accessible-by-www-ser) but there are [reports of `/tmp` having `755` permissions on some OSX installs](http://superuser.com/questions/753765/why-not-having-tmp-default-permissions-to-777-in-osx) which would put us back where we started.

Can we use [`Dir.tmpdir`](http://ruby-doc.org/stdlib-1.9.3/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir)?

**Comment by Hamms** [comment]
done

**Comment by islemaster** [comment]
:+1: LGTM!

**Pull request merged**